src/API/movies.js

```diff
@@ -40,4 +40,22 @@ export const UpdateMovie = async (obj) => {
         headers: { "Content-Type": "application/json" },
         body: JSON.stringify(obj),
     })
+}
+
+export const GetWatchlist = () => {
+    return fetch(`${url}/watchlist`)
+        .then((resp) => {
+            return resp.json();
+        })
+        .then((data) => {
+            return data;
+        });
+}
+
+export const AddToWatchlist = async (obj) => {
+    return await fetch(`${url}/watchlist/`, {
```

Comment:
since you are using async and await, add this in a try/catch block to capture errors

Reply:
Done